skimage/feature/blob.py

```diff
@@ -443,8 +443,8 @@ def blob_log(
         are given for each axis as a sequence, or as a single number, in
         which case it is equal for all axes.
     num_sigma : int, optional
-        The number of intermediate values of standard deviations to consider
-        between `min_sigma` and `max_sigma`.
+        The number of evenly-spaced values of standard deviations to consider
+        on the interval `[min_sigma, max_sigma]` (endpoints included).
```

Comment:
```suggestion
        The amount of standard deviations to consider in the evenly-spaced
        interval ``[min_sigma, max_sigma]``.
```
Restructured text needs double back-ticks for inline code. Do the other suggestions make sense to you? Feel very welcome to refine or reject. :)

Reply:
> Restructured text needs double back-ticks for inline code.

Right, and I recently got a refresher by @mdhaber himself! :pray:

Since `min_sigma` and `max_sigma` are parameter names, it could also be
```rst
[`min_sigma`, `max_sigma`]
```
if we (chose to) consider the square brackets as punctuation marks (and not strictly math symbols).

Reply:
> amount

sounds like something uncountable, so I prefer 'number of values.'

> evenly-spaced

I'm a huge fan of hyphenation, but we're not supposed to do it with adverbs, only adjectives; so it should be 'evenly spaced.'

I would write 'values of standard deviation' (plural for 'values' but not for 'standard deviation').

I would just write 'on the closed interval ``[min_sigma, max_sigma]``' because I think it's clear enough, by the meaning of 'closed interval' and by the notation with square brackets.